server/utils.py

```diff
@@ -17,3 +25,25 @@ def to_python(self, value):
 
     def to_url(self, value):
         return self.hashids.encode(value)
+
+def send_email(to, subject, body, link="http://okpy.org", linktext="Sign into okpy.org"):
+    """ Send an email using sendgrid.
+    Usage: send_email('student@okpy.org', 'Hey from OK', '<h1>hi</h1>')
+    """
+    html = render_template('email/base.html', to=to, subject=subject,
+                           body=body, link=link, link_text=linktext)
+    message = sendgrid.Mail(
+        to=to,
+        from_name="Okpy.org",
+        from_email="no-reply@okpy.org",
+        subject=subject,
+        html=transform(html),
+        text=body)
+
+    try:
+        status, msg = sg.send(message)
+        return status
+    except SendGridClientError as e:
+        log.info(exc_info=True)
```

Comment:
Should this be `error`, not `info`? Seems pretty serious if a notification doesn't get sent.